sqlmesh/core/renderer.py

```diff
@@ -307,7 +314,7 @@ def render(
         Returns:
             The rendered expression.
         """
-        cache_key = self._cache_key(start, end, execution_time)
+        cache_key = self._cache_key(start, end, execution_time, kwargs.get("evaluating"))
```

Comment:
Why not make it a top-level argument?

Reply:
Addressed in https://github.com/TobikoData/sqlmesh/pull/1562/commits/74f69aa9097c6c7904b2f4f6f5ebb620acea8600